### PR TITLE
Key shortcuts for increase/decrease/reset font size

### LINKED ISSUE #81: Increase font size with Ctrl & +
https://stackoverflow.com/questions/21900938/how-do-i-change-the-font-style-in-ace-editor#21909788

### PR DESCRIPTION
**Pull request by zuphilip**
This fixes #81.

### CODE REVIEW

**Comment by zuphilip** [comment]
(Only the menus don't resize)

File: src/chrome/content/scaffold/scaffold.js [hunk 1]
```diff
@@ -108,6 +111,12 @@ var Scaffold = new function() {
 		_editors["tests"].getSession().setUseSoftTabs(false);
 		
 		_editors["import"].getSession().setMode(new importWin.TextMode);
+		
+		// Set font size from previous session
+		var size = Zotero.Prefs.get("scaffold.fontSize");
```
Comment (zuphilip) [inline src/chrome/content/scaffold/scaffold.js]: @dstillman Please check whether it is okay to use Zotero preferences here.

Comment (zuphilip) [reply src/chrome/content/scaffold/scaffold.js]: Ping @dstillman

Comment (zuphilip) [reply src/chrome/content/scaffold/scaffold.js]: Okay, good idea with this function. Is the argument then also the same, i.e. `Zotero.Prefs.clear("scaffold.fontSize")`?

Comment (dstillman) [reply src/chrome/content/scaffold/scaffold.js]: Yep.

**Comment by zuphilip** [comment]
Please have a look at the new version. Should be ready for merge now.

File: src/chrome/content/scaffold/scaffold.js [hunk 2]
```diff
@@ -362,13 +360,18 @@ var Scaffold = new function() {
 				var typeNames = Zotero.ItemTypes.getTypes().map(t => t.name);
 				document.getElementById('output').value = JSON.stringify(typeNames, null, '\t');
 				break;
+			case "shortcuts":
+				var value = Zotero.File.getContentsFromURL(`chrome://scaffold/content/templates/shortcuts.txt`);
```
Comment (dstillman) [inline src/chrome/content/scaffold/scaffold.js]: Did you forget to commit this file?

Comment (zuphilip) [reply src/chrome/content/scaffold/scaffold.js]: Yes, sorry for that. Now, I add this missing file and solved the merge conflicts before. Please have a look at the new version.

**Comment by zuphilip** [comment]
@dstillman Can we merge this now or do you have anything else to consider?

(I work with a version of Scaffold including this patch for several weeks and it works fine.)

**Pull request merged**